src/main/scala/com/wavesplatform/settings/WavesSettings.scala

```diff
@@ -34,6 +35,7 @@ object WavesSettings {
     val dataDirectory           = config.as[String](s"$configPath.data-directory")
     val maxCacheSize            = config.as[Int](s"$configPath.max-cache-size")
     val maxRollbackDepth        = config.as[Int](s"$configPath.max-rollback-depth")
+    val rememberBlocks          = config.as[Option[Long]](s"$configPath.remember-blocks-interval-in-cache").getOrElse(120L * 60L * 1000L)
```

Comment:
`reference.conf` is *the* place for specifying defaults. Also this could be `FiniteDuration`